src/System.Management.Automation/engine/cmdlet.cs

```diff
@@ -49,10 +49,24 @@ public static HashSet<string> CommonParameters
         private static Lazy<HashSet<string>> s_commonParameters = new Lazy<HashSet<string>>(
             () =>
             {
-                return new HashSet<string>(StringComparer.OrdinalIgnoreCase) {
-                    "Verbose", "Debug", "ErrorAction", "WarningAction", "InformationAction",
-                    "ErrorVariable", "WarningVariable", "OutVariable",
-                    "OutBuffer", "PipelineVariable", "InformationVariable" };
+                if (!ExperimentalFeature.EnabledExperimentalFeatureNames.Contains("PSNewCommonParameters"))
+                {
+                    return new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+                    {
+                        "Verbose", "Debug", "ErrorAction", "WarningAction", "InformationAction",
+                        "ErrorVariable", "WarningVariable", "OutVariable",
+                        "OutBuffer", "PipelineVariable", "InformationVariable"
+                    };
+                }
+
+                // If you add new common parameters, manually sort them into this list
```

Comment:
It's really just a request, so that anyone who wants to see a list of common parameters using the public static property can see them in sorted order:

```powershell
[System.Management.Automation.Cmdlet]::CommonParameters
```

It is much easier to visually parse that information when it is sorted.

I have updated the comment by adding "please" since it's an ask, not a hard requirement, and stating why they are sorted into that list.